BusinessLogicLayer/Services/FileService.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using System.Threading.Tasks;
+using BusinessLogicLayer.DtoModels;
+using BusinessLogicLayer.Interfaces;
+using DataAccessLayer.Service;
+using DbMigrations.EntityModels;
+using AutoMapper;
+using BusinessLogicLayer.Mapping;
+
+namespace BusinessLogicLayer.Services
+{
+    public class FileService : IFileService
+    {
+        private readonly IUnitOfWork unitOfWork;
+        private readonly FileProfile profile;
```

Comment:
I think there will be some mistakes.
If i clearly remember should be: 
private readonly FileProfile profile = new FileProfile();
or
private readonly FileProfile profile = new();